torchvision/datasets/__init__.py

```diff
@@ -1,3 +1,4 @@
+from ._optical_flow import KittiFlow, Sintel
```

Comment:
Note the underscore: we may or may not make the `FlowDataset` base class public eventually, but for now it's safer to not expose anything